frontend/src/components/Search/Search.vue

```diff
@@ -160,6 +160,48 @@
 								@blur="lostFocusOnSearch"
 							/>
 							<AccountLink :address="contract.creator.asString" />
+							<div>
+								Age
+								<Tooltip
+									:text="formatTimestamp(contract.blockSlotTime)"
+									:position="index === 0 ? tooltipPositionBottom : ''"
```

Comment:
This line is repeated in all the `<Tooltip>` inputs. It would be great if it could be refactored so you don't have to re-type it everywhere.